libs/langchain/tests/integration_tests/document_loaders/test_pdf.py

```diff
@@ -136,3 +137,26 @@ def test_mathpix_loader() -> None:
     docs = loader.load()
     assert len(docs) == 1
     print(docs[0].page_content)
+
+# def test_amazontextract_loader() -> None:
```

Comment:
Should remove this before opening the PR, or replace the actual file path with a placeholder, and add a ` @pytest.mark.skip()` tag, so that users can still manually test.

Reply:
sure, we'll remove the tests. That is just for your reference so you can test more easily.